modules/openapi-generator/src/main/resources/typescript-angular/README.mustache

```diff
@@ -124,21 +124,23 @@ export class AppComponent {
 }
 ```
 
-Note: The ApiModule is restricted to being instantiated once app wide.
+Note: The {{apiModuleClassName}} is restricted to being instantiated once app wide.
 This is to ensure that all services are treated as singletons.
 
 #### Using multiple OpenAPI files / APIs / ApiModules
 In order to use multiple `ApiModules` generated from different OpenAPI files,
-you can create an alias name when importing the modules
-in order to avoid naming conflicts:
+you can avoid naming conflicts by configuring `apiModulePrefix` when generating other modules.
+
+Eg: Configuring another module using `apiModulePrefix` of `Other`:
+
 ```
-import { ApiModule } from 'my-api-path';
-import { ApiModule as OtherApiModule } from 'my-other-api-path';
+import { {{apiModuleClassName}} } from 'my-api-path';
+import { OtherApiModule } from 'my-other-api-path';
```

Comment:
please keep the alias example with `as`, see https://github.com/OpenAPITools/openapi-generator/issues/7722#issuecomment-709406844

Reply:
Done